### PR TITLE
Some manifest fields can now include variables

### PR DESCRIPTION
**Pull request by itowlson**
**This is for Spin 2.4.**

Content must go through a pre-merge checklist.

## Pre-Merge Content Checklist

This documentation has been checked to ensure that:

- [ ] The `title`, `template`, and `date` are all set
- [ ] Does this PR have a new menu item (anywhere in `templates/*.hbs` files) that points to a document `.md` that is set to publish in the future? If so please only publish the `.md` and `.hbs` changes in real-time (otherwise there will be a menu item pointing to a `.md` file that does not exist)
- [ ] File does not use CRLF, but uses plain LF (hint: use `cat -ve <filename> | grep '^M' | wc -l` and expect 0 as a result)
- [ ] Has passed [`bart check`](https://developer.fermyon.com/bartholomew/quickstart)
- [ ] Has been manually tested by running in Spin/Bartholomew (hint: use `PREVIEW_MODE=1` and run `npm run styles` to update styling)
- [ ] Headings are using Title Case
- [ ] Code blocks have the programming language set to properly highlight syntax and the proper copy directive
- [ ] Have tested with `npm run test` and resolved all errors
- [ ] Relates to an existing (potentially outdated) blog article? If so please add URL in blog to point to this content.

### CODE REVIEW

File: content/spin/v2/manifest-reference.md [hunk 1]
```diff
@@ -49,6 +50,20 @@ command = "cargo build --target wasm32-wasi --release"
 watch = ["src/**/*.rs", "Cargo.toml"]
 ```
 
+## Using Variables in the Manifest
+
+The following fields allow you to use [expressions](./variables.md#adding-variables-to-your-applications) in their values:
+
+* `application.trigger.redis.address`
```
Comment (vdice) [inline content/spin/v2/manifest-reference.md:57]: Wondering if we also want to cross-reference this info in applicable spots elsewhere in the docs?  eg
- https://developer.fermyon.com/spin/v2/redis-trigger#redis-trigger-application-settings for `application.trigger.redis.address`
- https://developer.fermyon.com/spin/v2/redis-trigger#specifying-a-redis-trigger for `trigger.redis.channel`
- https://developer.fermyon.com/spin/v2/http-outbound#granting-http-permissions-to-components for `allowed_outbound_hosts`.

Maybe a brief note around how variables can be used in these scenarios?

Comment (itowlson) [reply content/spin/v2/manifest-reference.md:57]: @vdice Yes, that's a great idea!  Thanks - will do!

**Review by vdice — APPROVED** [review]
LGTM!

_(shakes fist at cloud)_

**Comment by itowlson** [comment]
I've reviewed the CI errors and they relate to a change in the Python SDK that was not intended to affect the docs but did: the errors are not, as far as I can tell, related to this PR. So overriding to SMASH THAT MERGE BUTTON ON THE CORRECT PR THIS TIME

**Pull request merged**